airbyte-integrations/connectors/source-hubspot/source_hubspot/streams.py

```diff
@@ -807,6 +855,24 @@ def _process_search(
 
         return list(stream_records.values()), raw_response
 
+    def _read_associations(self, records: Iterable, identifiers: Iterable[Union[int, str]]) -> Iterable[Mapping[str, Any]]:
+        records_by_pk = {record[self.primary_key]: record for record in records}
+        identifiers = list(identifiers)
+        associations_stream = AssociationsStream(
+            api=self._api, start_date=self._start_date, credentials=self._credentials, parent_stream=self, identifiers=identifiers
+        )
+        slices = associations_stream.stream_slices(sync_mode=SyncMode.full_refresh)
+
+        for _slice in slices:
```

Comment:
Do we need to update the schema of the output objects?

Reply:
I don't think this should change the schema as this is only a bug fix to retrieve the associations in incremental mode.
I looked at the `deals.json` stream as an example and it already has the `companies`, `contacts`, and `line_items` fields

Reply:
Alexandre is right. Schemas already contain those fields, only data was missing in incremental mode